### ISSUE TITLE
Errors on startup without explanations, sth about fonts

### ISSUE THREAD

**Issue by michzimny** [up2university/multiparty-meeting#5]
I'm setting up a new instance. I'm using misi/mm:latest (`docker images` reports ID: 243eb2cd8f8b).

My directory layout:
```
$ find certs/
certs/
certs/.gitignore
certs/privkey.pem
certs/cert.pem
$ find configs/
configs/
configs/server
configs/server/config.js
configs/app
configs/app/config.js
$ cat run.sh
#/bin/bash
source .env
docker run \
        -v ${PWD}/configs/app:${BASEDIR}/${MM}/app/config \
        -v ${PWD}/configs/server:${BASEDIR}/${MM}/server/config \
        -v ${PWD}/certs:${BASEDIR}/${MM}/server/certs \
        -e BASEDIR=${BASEDIR} -e MM=${MM} \
        --network host \
        --restart unless-stopped \
        --name mm \
        --detach \
      misi/mm
```

In the config, I've only set rtcAnnouncedIPv4 and TURN settings.

When starting the image with Misi's run.sh, I'm getting:
```
Container starting up...
- process.env.DEBUG: undefined
- config.mediasoup.logLevel: warn
- config.mediasoup.logTags: [ 'info', 'ice', 'dtls', 'rtp', 'srtp', 'rtcp', 'rbe', 'rtx' ]
[opening Readline Command Console...]
cmd> [12:12:30] NODE_ENV: production
[12:12:30] Using gulpfile /opt/multiparty-meeting/app/gulpfile.js
[12:12:30] Starting 'dist-watch'...
[12:12:30] Starting 'clean'...
[12:12:30] 'clean' errored after 18 ms
[12:12:30] Error: EINVAL: invalid argument, rmdir '/opt/multiparty-meeting/server/public/resources/fonts'
[12:12:30] 'dist-watch' errored after 22 ms
/
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/chooseRoom.html'
/
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/chooseRoom.html'
/
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/chooseRoom.html'
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/index.html'
/
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/chooseRoom.html'
/
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/chooseRoom.html'
/
Error: ENOENT: no such file or directory, stat '/opt/multiparty-meeting/server/public/chooseRoom.html'
```

The page in the browser returns "Not Found" only.

How to track more details of these errors?

**Comment by misi** [comment]
See:
git diff 4db338..4eb355
e.g. You have a
+RUN git checkout tags/1.1

Would it be possible to put all customization in misi/mm? 
So not fork for up2u but customize from configs and volume mount everything.
I don't know if it is possible or not. I hope it is. 
It would be for me easier to update and you (and any other user) will also benefit of a customizable image..
I mean make misi/mm ready to any user can customize it very well, so give you the tools and hooks that you need for up2u.
In such case we could avoid such code skew.

**Comment by misi** [comment]
But this error is strange...
I will try to build your branch.
How did you build the image? `docker-compose up --build`?

I never had problem with the clean..

I have just rebuilt and tested misi/mm master branch.
I don't have such issue with it.

I fully understand that for you I guess more simple and preferred to use this forked model that we used with knockplop, but it may adds such complication. 

e.g. Recently as you wrote I moved and renamed config and cert dirs and so I changed in both multiparty-meeting and misi/mm the configs simultan. 
Maybe that is causing the issue? I am not sure...

But anyhow I think you should remove the line that checks out the branch 1.1.

**Comment by michzimny** [comment]
Yes, we can customize instances via configs and volume mounts as you said. That will be easier, sure.

But the issue here is rather not because of the forking model. I've not been building the image on my own. I have just pulled from misi/mm. So ,the `+RUN git checkout tags/1.1` should have not been a problem, but I just removed it, thanks.

Maybe I could give you the access to the problematic host to have a deeper look? Please send me your public SSH key.

**Comment by misi** [comment]
centos + docker issue..
https://github.com/moby/moby/issues/27214

> I finally found a solution by digging though the mess of related issues. Commenting here since this was my starting point, and maybe it will help someone else.
> 
> I installed the latest 4.12 kernel, and it didn't make a difference. This is actually a problem with the overlay fs in centos. The solution was to reformat the filesystem where the docker config is stored (/var/lib/docker) with some flags: mkfs -t xfs -n ftype=1 /PATH/TO/DEVICE
> 
> The problem is better described in #27358, but the solution is best explained in #29873.
> More details from RedHat

**Comment by michzimny** [comment]
@misi, how to reproduce the error from moby/moby#27214 on the host? I'm only seeing the errors I provided above.

**Comment by misi** [comment]
try to `docker exec -it mm bash` and try to remove a directory it will fail..
